web/cobrands/angus/layout.scss

```diff
@@ -0,0 +1,465 @@
+    color: $primary_b;
+}
+
+[class*=" icon-"]:before,[class^=icon-]:before
+{
+  -moz-osx-font-smoothing:grayscale;
+  -webkit-font-smoothing:antialiased;
+  font-family:angusgov;
+  font-style:normal;
+  font-variant:normal;
+  font-weight:400;
+  line-height:1;
+  speak:none;
+  text-transform:none;
+}
+
+.icon-facebook:before
+{
+  content:"\e61c";
+}
+
+.icon-twitter:before
+{
+  content:"\e61d";
+}
+
+.icon-youtube:before
+{
+  content:"\e61e";
+}
+
+footer p.social
+{
+  overflow:hidden;
+}
+
+footer p.social a
+{
+  -moz-transition:.15s color ease-in,.15s background-color ease-in;
+  -webkit-transition:.15s color ease-in,.15s background-color ease-in;
+  background-color:#3c3c3c;
+  border-radius:1em;
+  float:left;
+  font-size:2.25rem;
+  height:1em;
+  margin:0 .25em 0 0;
+  text-indent:-20000px;
+  transition:.15s color ease-in,.15s background-color ease-in;
+  width:1em;
+}
+
+footer p.social a:before
+{
+  float:left;
+  font-size:1.25rem;
+  line-height:1.8;
+  text-align:center;
+  text-indent:0;
+  width:100%;
+}
+
+.lt-ie9 footer p.social a:before
+{
+  font-size:1.25em;
+}
+
+footer p.social a:active,footer p.social a:focus,footer p.social a:hover
+{
+  background-color:#353535;
+  text-decoration:none;
+}
+
+.ie6, .ie7, .ie8 {
+  footer p.social a {
+    font-size:2.25em;
+  }
+}
+
+.ie6, .ie7 {
+  footer p.social a {
+    font-size:1.2em!important;
+    height:1.5em;
+    text-indent:0;
+    width:auto;
+  }
+}
+
+.wrapper__breadcrumb {
+  border-bottom: 1px solid $angus_grey_link;
+}
+
+.container__breadcrumb {
+  display: block;
+  font-size: 0.9em;
+  padding: 1em 20px .75em 20px;
+
+  ul {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    li {
+      display: inline;
+
+      &::before {
+        content: ">";
+        padding: 0 10px;
+      }
+      &:first-child::before {
+        content: "";
+      }
+
+      a {
+        color: $angus_dark_blue;
+      }
+    }
+  }
+}
+
+.nav-menu--main {
+  display: none;
+}
+
+
+.nav-menu--breadcrumb {
+  display: block;
+  font-size: 0.75em;
+  float: left;
+
+  a {
+    color: $angus_dark_blue;
+  }
+  span {
+    color: $angus_green;
+  }
+}
+
+.nav-menu--main {
+  a {
+    color: $angus_light_grey;
+  }
+
+  a:hover {
+    text-decoration: underline;
+  }
+
+  a.report-a-problem-btn {
+    color: $angus_light_grey;
+    background-color: $angus_light_green;
+  }
+
+  span {
+    color: white;
+  }
+}
+
+body.mappage {
+  .nav-menu--main {
+    display: block;
+  }
+  .nav-menu--breadcrumb {
+    display: none;
+  }
+
+  .wrapper__breadcrumb {
+    display: none;
+  }
+
+  #site-logo {
+    height: 54px;
+    width: 111px;
+    top: 7px;
+    left: 7px;
+  }
+
+  #site-header {
+    border-bottom: 0;
+    box-shadow: none;
+
+    form.form__search {
+      display: none;
+    }
+
+    .container {
+      min-height: 4em;
+    }
+  }
+
+  #user-meta {
+    p {
+      color: $primary_text;
+    }
+
+    & + .wrapper + .wrapper + .container .content {
```

Comment:
Would a generic sibling selector be okay here? Think that's supported from IE7 same as `+`.